packages/yaml/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@rollup/plugin-yaml",
+  "version": "2.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Convert YAML files to ES6 modules",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "rollup",
+  "homepage": "https://github.com/rollup/plugins/packages/yaml/#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "rollup -c",
+    "ci:coverage": "nyc pnpm run test && nyc report --reporter=text-lcov > coverage.lcov",
+    "ci:lint": "pnpm run build && pnpm run lint && pnpm run security",
+    "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
+    "ci:test": "pnpm run test -- --verbose",
+    "lint": "pnpm run lint:js && pnpm run lint:docs && pnpm run lint:package",
+    "lint:docs": "prettier --single-quote --write README.md",
+    "lint:js": "eslint --fix --cache src test",
+    "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
+    "prebuild": "del-cli dist",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm run lint",
+    "pretest": "npm run build",
```

Comment:
Looks like other plugins already in the repo also use `npm` for these scripts instead of `pnpm`, is that intentional? They seemed to work fine locally when I tested them with `pnpm`.

Reply:
good catch. this will need fixing in other plugins likely as well